### PR TITLE
Fix font error

### LINKED ISSUE #192: Fix font error

### PR DESCRIPTION
**Pull request by rosaigor138**
Closes #192

### CODE REVIEW

**Review by LBeghini — APPROVED** [review]
No errors found on MacOS. Mathematical symbols on equations page works normally.

**Comment by atilaajones** [comment]
Remember to insert the issue before merging.

**Review by atilaajones — CHANGES_REQUESTED** [review]
I don't think the font looks good, the mathematical symbols are crude. I recommend using Cambria Math, inserting the file in the source, as it is a specific font for mathematics.

**Comment by atilaajones** [comment]
If you change the font to _Cambria Math_ in Wizard Equations, the same change must be made to the equations in the PDF report.

File: source/view/wizard/pages/equations_page.py [hunk 1]
```diff
@@ -24,7 +27,8 @@ def __init__(self):
 
     def set_content_attributes(self):
         self.equation.setPlaceholderText("insert your equation here...")
-        self.equation.setFont(QFont("Cambria Math", 12))
+        self.equation.setFont(QFont("DejaVu Sans", 12))
```
Comment (LBeghini) [inline source/view/wizard/pages/equations_page.py:30]:
```suggestion
        self.equation.setFont(QFont("Cambria Math", 12))
```

Needs to be changed back to Cambria Math

File: source/view/wizard/pages/equations_page.py [hunk 2]
```diff
@@ -70,7 +74,7 @@ def __init__(self, update_line_text, dictionary):
             button = QPushButton(key)
             button.setText(key)
             button.setMinimumWidth(len(key))
-            button.setFont(QtGui.QFont("Cambria Math", 9))
+            button.setFont(QtGui.QFont("DejaVu Sans", 9))
```
Comment (LBeghini) [inline source/view/wizard/pages/equations_page.py:77]:
```suggestion
            button.setFont(QtGui.QFont("Cambria Math", 9))
```
Needs to be changed back to Cambria Math

File: source/domain/pdf_model.py
Comment (LBeghini) [inline source/domain/pdf_model.py:1]: In this file, the "Deja Vu Sans" font should be changed to "Cambria Math"

**Review by LBeghini — CHANGES_REQUESTED** [review]
The idea is to also remove the DejaVu Sans font from files, and all its references.

File: source/view/wizard/pages/equations_page.py (hunk 2 above)
Comment (rosaigor138) [reply source/view/wizard/pages/equations_page.py]: I changed the font file we were using to Libertinus Math

**Review by atilaajones — CHANGES_REQUESTED** [review]
The invariant buttons and the equation text field are faulty, with horizontal lines (simulate a failed laser printer). The PDF is right. See the images below. The printout isn't showing exactly how it looks on the screen.

image
image
image

**Comment by atilaajones** [comment]
The refactoring will cover all the code, and if the problem remains, it will be reconsidered.

**Pull request closed without merging**